packages/@aws-cdk/aws-lambda-event-sources/lib/kafka.ts

```diff
@@ -107,12 +110,14 @@ export class ManagedKafkaEventSource extends StreamEventSource {
         eventSourceArn: this.innerProps.clusterArn,
         startingPosition: this.innerProps.startingPosition,
         // From https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html#msk-password-limitations, "Amazon MSK only supports SCRAM-SHA-512 authentication."
```

Comment:
please move this comment to the new location as well.

Reply:
Moved